app/models/concerns/verification.rb

```diff
@@ -57,4 +57,8 @@ def no_phone_available?
   def sms_code_not_confirmed?
     !sms_verified?
   end
+
+  def user_type
+    level_three_verified? ? :level_3_user : level_two_verified? ? :level_2_user : :level_1_user 
```

Comment:
El operador ternario está bien cuando hay una sola condición y las tres partes son sencillas. Encadenar operadores ternarios así es un poco "abusar" :). Sugiero implementarlo utilizando `case`, como en [el tercer ejemplo de este artículo](http://www.skorks.com/2009/08/how-a-ruby-case-statement-works-and-what-you-can-do-with-it/)